File: Metastoneweb/src/Entity/Seance.php
<?php

namespace App\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Seance
 *
 * @ORM\Table(name="seance", indexes={@ORM\Index(name="seance_id_coach", columns={"id_coach"})})
 * @ORM\Entity
 */
class Seance
{
    /**
     * @var int
     *
     * @ORM\Column(name="Id_seance", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $idSeance;

    /**
     * @var \DateTime
     *
     * @ORM\Column(name="Date", type="date", nullable=false)
     */
    private $date;

    /**
     * @var \DateTime
     *
     * @ORM\Column(name="Duree", type="time", nullable=false)
     */
    private $duree;

    /**
     * @var int
     *
     * @ORM\Column(name="Nbr_participants", type="integer", nullable=false)
     */
    private $nbrParticipants;

    /**
     * @var string
     *
     * @ORM\Column(name="URL", type="string", length=255, nullable=false)
     */
    private $url;

    /**
     * @var float
     *
     * @ORM\Column(name="Prix", type="float", precision=10, scale=0, nullable=false)
     */
    private $prix;

    /**
     * @var \Utilisateur
     *
     * @ORM\ManyToOne(targetEntity="Utilisateur")
     * @ORM\JoinColumns({
     *   @ORM\JoinColumn(name="id_coach", referencedColumnName="ID_utilisateur")
     * })
     */
    private $idCoach;

    public function getIdSeance(): ?int
    {
        return $this->idSeance;
    }

    public function getDate(): ?\DateTimeInterface
    {
        return $this->date;
    }

    public function setDate(\DateTimeInterface $date): self
    {
        $this->date = $date;

        return $this;
    }

    public function getDuree(): ?\DateTimeInterface
    {
        return $this->duree;
    }

    public function setDuree(\DateTimeInterface $duree): self
    {
        $this->duree = $duree;

        return $this;
    }

    public function getNbrParticipants(): ?int
    {
        return $this->nbrParticipants;
    }

    public function setNbrParticipants(int $nbrParticipants): self
    {
        $this->nbrParticipants = $nbrParticipants;

        return $this;
    }

    public function getUrl(): ?string
    {
        return $this->url;
    }

    public function setUrl(string $url): self
    {
        $this->url = $url;

        return $this;
    }

    public function getPrix(): ?float
    {
        return $this->prix;
    }

    public function setPrix(float $prix): self
    {
        $this->prix = $prix;

        return $this;
    }

    public function getIdCoach(): ?Utilisateur
    {
        return $this->idCoach;
    }

    public function setIdCoach(?Utilisateur $idCoach): self
    {
        $this->idCoach = $idCoach;

        return $this;
    }


}

File: Coaching/src/entities/Seance.java
package entities;


import java.sql.Time;
import java.sql.Date;

/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

/**
 *
 * @author Aziz
 */
public class Seance {
    int ID_Seance;
    int ID_coach;
    Date date;
    Time duree;
    int NBR_Participant;
    String URL;
    float prix;

    public Seance(int ID_Seance, int ID_coach, Date date, Time duree, int NBR_Participant, String URL, float prix) {
        this.ID_Seance = ID_Seance;
        this.ID_coach = ID_coach;
        this.date = date;
        this.duree = duree;
        this.NBR_Participant = NBR_Participant;
        this.URL = URL;
        this.prix = prix;
    }

    public int getID_Seance() {
        return ID_Seance;
    }

    public int getID_coach() {
        return ID_coach;
    }

    public Date getDate() {
        return date;
    }

    public Time getDuree() {
        return duree;
    }

    public int getNBR_Participant() {
        return NBR_Participant;
    }

    public String getURL() {
        return URL;
    }

    public float getPrix() {
        return prix;
    }

    public void setID_Seance(int ID_Seance) {
        this.ID_Seance = ID_Seance;
    }

    public void setID_coach(int ID_coach) {
        this.ID_coach = ID_coach;
    }

    public void setDate(Date date) {
        this.date = date;
    }

    public void setDuree(Time duree) {
        this.duree = duree;
    }

    public void setNBR_Participant(int NBR_Participant) {
        this.NBR_Participant = NBR_Participant;
    }

    public void setURL(String URL) {
        this.URL = URL;
    }

    public void setPrix(float prix) {
        this.prix = prix;
    }

    @Override
    public String toString() {
        return "Seance{" + "ID_Seance=" + ID_Seance + ", ID_coach=" + ID_coach + ", date=" + date + ", duree=" + duree + ", NBR_Participant=" + NBR_Participant + ", URL=" + URL + ", prix=" + prix + '}';
    }
    
}
